Sources/NIOCore/CurrentEventLoop.swift

```diff
@@ -0,0 +1,83 @@
+public struct CurrentEventLoop {
+    public let wrapped: EventLoop
+    @inlinable internal init(_ eventLoop: EventLoop) {
+        self.wrapped = eventLoop
+    }
+}
+
+#if swift(>=5.6)
+@available(*, unavailable)
+extension CurrentEventLoop: Sendable {}
```

Comment:
Add some documentation on it why it is not sendable - not everyone may understand this unavailable trick :)